commit ae439aa1d5ab04b1eeaf3de549248d193f9670b1
Author: illiliti <illiliti@protonmail.com>
Date:   Sun Jul 3 18:14:15 2022 +0300

    meson: use install_emptydir to create directory
    
    instead of install_subdir. Fixes muon - a strictly-conforming meson
    implementation which doesn't implement deprecated and broken-by-design
    functionality.
    
    For more info, see: https://mesonbuild.com/Reference-manual_functions.html#install_subdir
    
    Signed-off-by: illiliti <illiliti@protonmail.com>

diff --git a/meson.build b/meson.build
index 160f39cb..4324a752 100644
--- a/meson.build
+++ b/meson.build
@@ -329,7 +329,7 @@ libquirks = static_library('quirks', src_libquirks,
 dep_libquirks = declare_dependency(link_with : libquirks)
 
 # Create /etc/libinput
-install_subdir('libinput', install_dir : dir_etc)
+install_emptydir(dir_etc / 'libinput')
 
 ############ libinput.so ############
 install_headers('src/libinput.h')
